srcpkgs/leftwm/INSTALL.msg

```diff
@@ -0,0 +1,9 @@
+Configuring
```

Comment:
There is no reason, why the package manager should inform you about this type of documentation.
This is a tiling WM, you are usually not using it without reading the documentation at all, and you provide the `README.md` in the package which already has the information about `config.toml`